apis/Google.Cloud.Storage.V1/Google.Cloud.Storage.V1/RetryHandler.cs

```diff
@@ -52,11 +52,14 @@ internal static async Task<bool> IsRetriableResponse(HttpResponseMessage respons
                 }
 
                 // For other errors, check the response content.
-                var str = await response.Content.ReadAsStringAsync().ConfigureAwait(false);
-                errorResponse = NewtonsoftJsonSerializer.Instance.Deserialize<StandardResponse<object>>(str);
-                var errors = errorResponse.Error?.Errors;
-                // If we couldn't deserialize properly, or the result doesn't contain any reasons, we won't retry.
-                return errors?.All(singleError => s_retriableErrors.Contains(singleError.Reason)) ?? false;
+                var json = await response.Content.ReadAsStringAsync().ConfigureAwait(false);
+                errorResponse = NewtonsoftJsonSerializer.Instance.Deserialize<StandardResponse<object>>(json);
+                return s_retriableErrorCodes.Contains(errorResponse.Error.Code);
+            }
+            catch (WebSocketException wsex)
```

Comment:
I think this needs a comment, at least. Are we convinced this is the exception thrown on all platforms?